lib/CSSStyleDeclaration.test.js

```diff
@@ -9,6 +9,7 @@ var parsers = require('./parsers');
 
 var dashedProperties = [...allProperties, ...allExtraProperties];
 var allowedProperties = dashedProperties.map(parsers.dashedToCamelCase);
+allowedProperties.push('cssFloat');
```

Comment:
```suggestion
var allowedProperties = ['cssFloat', ...dashedProperties.map(parsers.dashedToCamelCase)];
```

Reply:
The downside of your approach is that it creates an intermediate array in the `map` function, which is then immediately iterated over and discarded, thus having a `O(n) + O(n)` complexity, because it calls `map` and `[Symbol.iterator]`, whereas mine has `O(n) + O(1)` complexity, because it calls `map` and `push`.

---

The way your approach would remain `O(n)` would be to use:
```suggestion
var allowedProperties = ['cssFloat', ...dashedProperties.values().map(parsers.dashedToCamelCase)];
```

But that won't work until [**Iterator Helpers**](https://github.com/tc39/proposal-iterator-helpers) are implemented.